airgun/views/host.py

```diff
@@ -226,7 +226,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixinPF4):
         column_widgets={
             0: Checkbox(locator=".//input[@class='host_select_boxes']"),
             'Name': Text(
-                ".//a[contains(@href, '/new/hosts/') and not(contains(@href, 'Insights'))]"
+                "//a[contains(@href, '/new/hosts/') and not(contains(@href, 'Insights'))]"
```

Comment:
`//` will search in whole xml tree, can you please try `./`

Reply:
i tried with this ` ./` but it 's not workig